project/AppDependencies.scala

```diff
@@ -32,15 +32,15 @@ object AppDependencies {
   ) ++ test
 
   lazy val hmrcMongoTestPlay27: Seq[ModuleID] = Seq(
-    "org.scalatest"         %% "scalatest"       % "3.0.8", // version chosen for compatibility with scalatestplus-play
-    "com.vladsch.flexmark"  %  "flexmark-all"    % "0.35.10",
-    "org.mockito"           %% "mockito-scala"   % "1.10.1" % Test
+    "org.scalatest"         %% "scalatest"       % "3.0.8"   % Test, // version chosen for compatibility with scalatestplus-play
```

Comment:
This is required in compile time for `hmrc-mongo-test` and should be provided transitively.
However it does look like `hmrc-mongo-metrix` may be adding `hmrc-mongo-test` as a compile dependency rather than a test one...